cmd/forwarder/fixer.go

```diff
@@ -41,6 +41,19 @@ func getMetadata(req *http.Request, metadataId string) ([]byte, bool) {
 				metadataWriter.WriteString("\"")
 			}
 		}
+
+		// Add metadata about the credential if it is deprecated
+		if cred != nil && cred.Deprecated == true {
+			if !foundMetadata {
+				metadataWriter.WriteString("[")
+				metadataWriter.WriteString(metadataId)
+				foundMetadata = true
+			}
+			metadataWriter.WriteString(" fields=\"{'credential_deprecated': true, 'credential_name': '")
```

Comment:
Does this need to be escaped?

Reply:
TBH - yes. I got lazy. This is a bit tricky, though. We're sticking JSON data into a [PARAM-VALUE](https://tools.ietf.org/html/rfc5424#section-6). I'm also hoping to avoid too much conversion of structs to and from JSON for the sake of efficiency (this might happen alot).

One thing to note: It's unlikely anybody will have anything funky in the names of their credentials. These are the "usernames" parts of the auth. 

All that said - what do you think? I could do the work, or we can note that folks shouldn't use special characters in their user names. I need to think about exactly how to do the escaping here if I'm gonna do it.